sdcm/utils/raft/common.py

```diff
@@ -138,10 +138,11 @@ def clean_and_restart_bootstrap_after_abort(self):
         self.bootstrap_node.stop_scylla_server(ignore_status=True, timeout=600)
         # Clean garbage from group 0 and scylla data and restart setup
         try:
-            if self.verification_node.raft.get_diff_group0_token_ring_members():
```

Comment:
should first check the feature is there, before applying a different logic ?

Reply:
fixed